### PR TITLE
STAR-255-Switch-component-sizes

### PR DESCRIPTION
**Pull request by KDarcey**
### Purpose 🚀
To adjust the sizes of the Paprika Switch component so that it is consistent with the UI Kit.

### Notes ✏️
N/A

### Updates 📦
- [x] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/STAR-255-Switch-component-sizes

### Screenshots 📸
Before:
image

After:
image

### References 🔗
N/A

### CODE REVIEW

**Comment by jamiek-acl** [comment]
hey @KaanDarcey please note in the details of this PR you had this marked as a `major - breaking change`, but i changed it to `patch - bug fix`. we use these to determine versions of the releases.

File: packages/Switch/src/Switch.styles.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const KnobPositionStyles = {
     transform: translateX(12px);
   `,
   medium: css`
-    transform: translate(20px, 0);
+    transform: translate(16px, 0);
```
Comment (jamiek-acl) [inline packages/Switch/src/Switch.styles.js:11]: Normally we use `$space` token, or `stylers.spacer` helper... but these seem pretty arbitrary so i think how you did it is fine.

Comment (KDarcey) [reply packages/Switch/src/Switch.styles.js:11]: I could change that. Should i replace all of the width and height values with spacers as opposed to px as well?

Comment (jamiek-acl) [reply packages/Switch/src/Switch.styles.js:11]: I dont think so, as some would be like 2 spacer + 1/2 spacer

**Pull request merged**